Makefile

```diff
@@ -58,9 +62,9 @@ image: _builder
 run:
 	@echo "Running the MicroShift container"
 	sudo modprobe openvswitch
+	$(MAKE) _topolvm_create
 	sudo podman run --privileged --rm -d \
 		--name "${USHIFT_IMAGE}" \
-		--volume /dev:/dev:rslave \
```

Comment:
removing this mount will make the topoLVM not functional when creating volumes  (even thought all the pods will come up)